docs/kb/semgrep-appsec-platform/saml-google-workspace.md

```diff
@@ -0,0 +1,26 @@
+---
+description: Learn how to set up SAML access to Semgrep AppSec Platform with Google Workspace.
+tags:
+  - Semgrep AppSec Platform
+  - SSO
+---
+
+# SAML Single Sign-on with Google Workspace
+
+The process of setting up SAML Single Sign-on for the Semgrep AppSec Platform with Google Workspace is fundamentally similar to the process for other providers. However, there are a few quirks of Google Workspace to account for.
```

Comment:
Maybe this could just be something like, "This article demonstrates how to set up SAML Single Sign-on for Semgrep AppSec Platform with Google Workspace." I don't think we need to call out that there are quirks since we're pretty explicit in this article on how to set this up (I don't have strong opinions on this, though)

Reply:
That's fair. When I wrote the intro I think I hadn't decided if I was going to write a complete guide or not, but then I did. :)